md/single-sign-on-with-saml.md

```diff
@@ -206,8 +206,8 @@ Then you can try to access a portal page, an app page or a form URL (or just `ht
 Note that if you try to access `http://<bundle host>:<port>/bonita/login.jsp`, then you won't be redirected as this page still needs to be accessible in order for the tenant administrator (or another user if you set the property `saml.auth.standard.allowed` to true) to be able to log in without an account on the Identity Provider.
 
 ::: info
-**Note 1:** The single logout SAML profile is not supported by bonita as it doesn't work the same way for each IdP.  
-So the SingleLogoutService configuration is not used (but the element still needs to be present in order for the filter to work...).  
+**Note 1:** The Single Logout SAML profile is only provided as an experimental feature as it doesn't work the same way for each IdP.  
+Single logout has only been tested with a Keycloack server acting as SAML Identity Provider.   
```

Comment:
IMHO Might be worth to add something like "It's deactivated by default" or "deactivate it if you face problems"

Reply:
done